src/app/components/core/scheduler/list-schedules/list-schedules.component.ts

```diff
@@ -107,7 +107,7 @@ export class ListSchedulesComponent implements OnInit {
           this.ngProgress.done();
           // north_c - North C tasks, north_C - Northbound C Services
           data.schedules.forEach(sch => {
-            if (!['south_c', 'north_c', 'north_C', 'south', 'north', 'notification_c'].includes(sch.processName)) {
+            if (!['south_c', 'north_c', 'north_C', 'south', 'north', 'notification_c', 'automation_script'].includes(sch.processName)) {
```

Comment:
Add a comments which we ignored to filter with a descripition next to it

e.g. dispatcher_c - The Dispatcher service , management - ...